src/extras/core/ShapePath.js

```diff
@@ -13,6 +13,7 @@ function ShapePath() {
 	this.type = 'ShapePath';
 
 	this.color = new Color();
+	this.opacity = 1.0;
```

Comment:
If a new property to `ShapePath` is added, it's necessary to enhance the TS declaration file as well the docs.

Reply:
Uh, missed it in https://github.com/mrdoob/three.js/pull/19146/commits/88f399263d2718975f94fd5d2ca1cdc30db9e651 , removed.